scanner/src/main/kotlin/LocalScanner.kt

```diff
@@ -299,7 +300,7 @@ abstract class LocalScanner(
 
             if (missingArchives.isNotEmpty()) {
                 val pkgDownloadDirectory = downloadDirectory.resolve(pkg.id.toPath())
-                Downloader.download(pkg, pkgDownloadDirectory)
+                Downloader(DownloaderConfiguration()).download(pkg, pkgDownloadDirectory)
```

Comment:
Please mention in the commit message that this place, i.e. the scanners, will be made to adhere to the actual downloader config in the next commit.